odoo/addons/base/tests/test_views.py

```diff
@@ -4564,6 +4564,14 @@ def test_uncommented_invisible_field(self):
             'stock_intrastat',
             'sale_timesheet_enterprise',
             'helpdesk_repair',
+            'l10n_ch',
```

Comment:
![Selection_3858](https://github.com/user-attachments/assets/9b98cdaf-19a7-4f14-818b-bf2ec6db29d8) 😄

Reply:
Yeah I know, the thing is that those are not new addons, they have been there before this test. To my understanding, it's just that some addons were forgotten while creating the whitelist in the first place.